config/crd/bases/core.openfeature.dev_featureflags.yaml

```
@@ -0,0 +1,81 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: featureflags.core.openfeature.dev
+spec:
+  group: core.openfeature.dev
+  names:
+    kind: FeatureFlag
+    listKind: FeatureFlagList
+    plural: featureflags
+    shortNames:
+    - ffc
```

```suggestion
    - ff
```
I don't think ffc makes sense anymore with our new name.

---

There's a todo here - https://github.com/open-feature/open-feature-operator/pull/535/files#diff-c29b273426465350caa0db6a338509afcf85b7400bccfffd4bec81465c2e8c41R60 

should be gone when alpha support is removed

---

Exactly, this if we change it to `ff`, we will get CRD installation problems, as 2 CRDs have the same shortcut name